From: Felix Fietkau <nbd@nbd.name>
Date: Mon Jul 24 13:29:13 CEST 2023
Subject: [PATCH] mips: ralink: increase highmem start

Increase highmem start address in order to support devices with 512 MB memory
and MT7621 SoC

Signed-off-by: Felix Fietkau <nbd@nbd.name>
---
--- a/arch/mips/include/asm/mach-ralink/spaces.h
+++ b/arch/mips/include/asm/mach-ralink/spaces.h
@@ -5,6 +5,7 @@
 #define PCI_IOBASE	_AC(0xa0000000, UL)
 #define PCI_IOSIZE	SZ_16M
 #define IO_SPACE_LIMIT	(PCI_IOSIZE - 1)
+#define HIGHMEM_START   _AC(0x24000000, UL)
 
 #include <asm/mach-generic/spaces.h>
 #endif
